merge_data/shop_FR.mapping.json

```
@@ -468,12 +468,16 @@
       {
         "shop": "underwear"
       },
+      {
+        "shop": "clothes",
+        "clothes": "underwear"
+      },
       {
         "shop": "sewing"
       }
     ],
     "generate": {
-      "shop": "fabric;wool;household_linen;underwear;sewing",
+      "shop": "fabric;wool;household_linen;underwear;clothes;sewing",
```

As deprecated, we should remove `underwear` form produced values?
```suggestion
      "shop": "fabric;wool;household_linen;clothes;sewing",
```